themes-default/slim/views/layouts/main.mako

```diff
@@ -62,7 +62,12 @@
             % endif
             <%include file="/partials/alerts.mako"/>
                <div id="content-row" class="row">
-                    <div id="content-col" class="${'col-lg-10 col-lg-offset-1 col-md-10 col-md-offset-1' if not app.LAYOUT_WIDE else 'col-lg-12 col-md-12'} col-sm-12 col-xs-12">
+                    <div v-if="globalLoading" class="text-center ${'col-lg-10 col-lg-offset-1 col-md-10 col-md-offset-1' if not app.LAYOUT_WIDE else 'col-lg-12 col-md-12'} col-sm-12 col-xs-12">
```

Comment:
Could we maybe incorporate this as a separate component?

Reply:
I could try,
but I don't think it's worth the effort because we don't plan on keeping it.

Reply:
Why not keep it? You always need to load data right?

Reply:
I think it will be much less noticeable when it's a true single page application.